rider/views.py

```diff
@@ -17,7 +17,7 @@
 
 def list_group_view(request, r_id):
 	#Check that the rider exists
-	rider_exists_test = Rider.objects.filter(id=r_id)
```

Comment:
Oh it's not anything on you. They just called a variable 'rider_exists_test' and set it to an object, not its 'exists' value.